text/0060-tasks-api.md

```diff
@@ -500,34 +594,121 @@ This part demonstrates keyset paging in action on `/tasks`. The items `uid` rema
+| type      | string | No       | Permits to filter tasks by their related type. By default, when `type` query parameter is not set, all task types are concerned. It's possible to specify several types by separating them with the `,` character.                       |
+| beforeEnqueuedAt | string | No       | Filter tasks based on their enqueuedAt time. Retrieve tasks enqueued before the given filter value.              |
+| afterEnqueuedAt | string | No       | Filter tasks based on their enqueuedAt time. Retrieve tasks enqueued after the given filter value.  |
+| beforeStartedAt | string | No       | Filter tasks based on their startedAt time. Retrieve tasks started before the given value.                |
+| afterStartedAt | string | No       | Filter tasks based on their startedAt time. Retrieve tasks started after the given filter value.                    |
+| beforeFinishedAt | string | No       | Filter tasks based on their finishedAt time. Retrieve tasks finished before the given filter value.  |
+| afterFinishedAt | string | No       | Filter tasks based on their finishedAt time. Retrieve tasks finished after the given filter value.                 |
+
+##### 11.2. Query Parameters Behaviors
+
+###### 11.2.1. `uid`
 
-##### 10.2. Usages examples
+- Type: String
+- Required: False
+- Default: `*`
+
+`uid` is **case-unsensitive**.
+
+- 🔴 Sending values with a different type than `Integer` being separated by `,` for the `uid` parameter returns an [`invalid_task_uid`](https://github.com/meilisearch/specifications/blob/main/text/0061-error-format-and-definitions.md#invalid_task_uid)) error.
+
+
+###### 11.2.2. `indexUid`
+
+- Type: String
+- Required: False
+- Default: `*`
+
+`indexUid` is **case-sensitive**.
+
+###### 11.2.3. `status`
+
+- Type: String
+- Required: False
+- Default: `*`
+
+- 🔴 If the `status` parameter value is not consistent with one of the task statuses, an [`invalid_task_status`](0061-error-format-and-definitions.md#invalidtaskstatus) error is returned.
+
+###### 11.2.4. `type`
+
+- Type: String
+- Required: False
+- Default: `*`
+
+`type` is **case-insensitive**.
+
+- 🔴 If the `type` parameter value is not consistent with one of the task types, an [`invalid_task_type`](0061-error-format-and-definitions.md#invalidtasktype) error is returned.
+
+###### 11.2.5. Date Parameters
+
+###### 11.2.5.1. `beforeEnqueuedAt` and `afterEnqueuedAt`
+
+- Type: String
+- Required: False
+- Default: `*`
+
+###### 11.2.5.2. `beforeStartedAt` and `afterStartedAt`
+
+- Type: String
+- Required: False
+- Default: `*`
+
+###### 11.2.5.3. `beforeFinishedAt` and `afterFinishedAt`
+
+- Type: String
+- Required: False
+- Default: `*`
+
+###### 11.2.5.4. Date Format
+
+The filter accepts the RFC 3339 format. The following syntaxes are valid:
+
+- `Y-M-D`
+- `Y-M-DTH:M:SZ`
+- `Y-M-DTH:M:S+01:00`
```

Comment:
Not sure about that. @loiclec can you confirm that we support those 3 formats, please?

Reply:
There are two accepted syntaxes:
- `YYYY-MM-DD` (note the mandatory number of digits for year/month/day)
- a `date-time` defined by the "Internet Date/Time format" defined by RFC 3339 section 5.6: https://www.rfc-editor.org/rfc/rfc3339#section-5.6, which IMO we should not attempt to describe in detail in the specifications. But yes, in short it is something like `YYYY-MM-DDTHH-MM:SS` followed by an optional `.[0-9]+` followed by either `Z` or an offset `(+|-)HH:MM`

Reply:
I've updated the spec to make it more valid regarding the accepted formats without going into details. A suggestion is welcome if you have ideas on how to describe this more precisely, as always!